### ISSUE TITLE
Loss not coming down from 0.8 even after training for 5k steps using ssdlite_mobilenet_v2 

### ISSUE THREAD

**Issue by CHANDL3** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#244]
I am using my custom dataset for the training with 900 images(700 in train and 200 in test). After training  the ssdlite_mobilenet_v2_coco model for 3k steps the loss came down to the range of 1 to 2. Since then the model is not converging and the loss seems to vary between 0.8 and 1.5. Now the model got trained over 5k steps still the loss shows the same behavior. Can anybody help me out to figure out the reason.

**Comment by Petros626** [comment]
> I am using my custom dataset for the training with 900 images(700 in train and 200 in test). After training the ssdlite_mobilenet_v2_coco model for 3k steps the loss came down to the range of 1 to 2. Since then the model is not converging and the loss seems to vary between 0.8 and 1.5. Now the model got trained over 5k steps still the loss shows the same behavior. Can anybody help me out to figure out the reason.

A loss under 2 is recommended from EdjeElctronics so I think it's fine
